pkg/monitoring/metrics/infrastructure_metrics.go

```diff
@@ -0,0 +1,41 @@
+package metrics
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	ioprometheusclient "github.com/prometheus/client_model/go"
+)
+
+const (
+	singleStackIPv6True = 1.0
+)
+
+var (
+	infrastructureMetrics = []operatormetrics.Metric{
+		singleStackIpv6,
+	}
+
+	singleStackIpv6 = operatormetrics.NewGauge(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_hco_single_stack_ipv6",
+			Help: "Indicates whether the underlying cluster is single stack IPv6 (1) or not (0)",
+		},
+	)
+)
+
+// SetHCOMetricSingleStackIPv6True sets the gauge to 1 (true)
+func SetHCOMetricSingleStackIPv6True() {
+	singleStackIpv6.Set(singleStackIPv6True)
+}
+
+// IsHCOMetricSingleStackIPv6 returns true if underlying OCP cluster is single stack IPv6
+func IsHCOMetricSingleStackIPv6() (bool, error) {
+	dto := &ioprometheusclient.Metric{}
+	err := singleStackIpv6.Write(dto)
+	value := dto.Gauge.GetValue()
+
+	if err != nil {
+		return false, err
+	}
+
+	return value == singleStackIPv6True, nil
+}
```

Comment:
This method is not in use. Please remove.

Reply:
Also in the past it was not used, strange.
Removing.